### PR TITLE
Papercut

### PR DESCRIPTION
**Pull request by zimbatm**
Consistency and display fixes. /cc @rumblesan

### CODE REVIEW

File: lib/cide.rb [hunk 1]
```diff
@@ -195,7 +175,7 @@ def clean
       days_to_keep = options[:days]
       max_images = options[:count]
 
-      x = run('docker images --no-trunc', capture: true)
+      x = docker('images', '--no-trunc', capture: true)
```
Comment (rumblesan) [inline lib/cide.rb]: single letter variable feels smelly

File: lib/cide.rb [hunk 2]
```diff
@@ -208,7 +188,7 @@ def clean
         return
       end
 
-      x = run("docker inspect #{cide_image_ids.join(' ')}", capture: true)
+      x = docker('inspect', *cide_image_ids, capture: true)
```
Comment (rumblesan) [inline lib/cide.rb]: here as well

**Comment by rumblesan** [comment]
so it all looks good to me. comments are really nit picky, but they're the only vaguely problematic things I could find

File: lib/cide.rb (hunk 2 above)
Comment (zimbatm) [inline lib/cide.rb]: fair. since these changes don't introduce the x variable i'm going to let that for a future change

**Pull request merged**